api/sink/sink.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package nighthawk.sink;
```

Comment:
In our last discussion about namespaces, we ended up on having just a single namespace for Nighthawk's future code. Do we have a good reason to introduce the new namespace here?

Reply:
Modified in 5c49c607558f7fb26fb088c787b8cb22dd820e5d